package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": false,
   "scripts": {
-    "dev": "next dev",
+    "dev": "NODE_OPTIONS='--inspect' next dev",
```

Comment:
Can we keep the original `dev` command and just add the new one as `dev:debug` please.